workspaces/ui/src/App.js

```diff
@@ -66,30 +66,34 @@ function AppError() {
 
         <h5>
           Please feel free to{' '}
-          <a href="mailto:hi@useoptic.com?subject=Optic App crash report">
+          <a href="mailto:aidan@useoptic.com?subject=Optic App crash report">
             contact us about it.
           </a>
         </h5>
 
         <p>Meanwhile, you could try the following:</p>
 
         <ul>
+          <li>
+            Create a debug file and{' '}
+            <a href="mailto:aidan@useoptic.com?subject=Optic App crash report">
+              {' '}
+              email it to our team
+            </a>
+            or{' '}
+            <a href="https://github.com/opticdev/optic/issues">
+              {' '}
+              open a GitHub issue
+            </a>{' '}
+            Just open Developer Tools and type `debugOptic()`.{' '}
+            <a target="_blank">Read more here</a>
+          </li>
           <li>
             <a href="#" onClick={onClickRefresh}>
               Refresh the page
             </a>{' '}
             in your browser.
           </li>
-          <li>
-            Try your browser in{' '}
-            <a
-              href="https://en.wikipedia.org/wiki/Privacy_mode"
-              target="_blank"
-            >
-              "privacy" or "incognito" mode
-            </a>
-            .
-          </li>
```

Comment:
Removed some legit cache-busting / broken cookie / local storage / service worker advice! Not that we have any of that, at the moment 😅

Reply:
Yeah since it probably wouldn't help I didn't want to give them a false path to go down, esp if it destroyed their current session (which I wanted them to provide) @JaapRood